### PR TITLE
app-text/mupdf-1.21.1: fix url processing and fix QA warning

### PR DESCRIPTION
**Pull request by rndxelement**
In <=app-text/mupdf-1.21.1 built with USE=opengl trying to open
an URL in a PDF file results in "warning: cannot spawn browser
'xdg-open': No such file or directory".
Apply fix from upstream to resolve the issue until the next
release of MuPDF.
See: [Bug](https://bugs.gentoo.org/893604)
See: [Upstream commit](https://github.com/ArtifexSoftware/mupdf/commit/37757db262425d793b17b63821d9014d3655e50a) 

Also fix QA warning "excessiveLineLength" on line 160.

### CODE REVIEW

File: app-text/mupdf/mupdf-1.21.1.ebuild [hunk 1]
```diff
@@ -54,6 +54,9 @@ PATCHES=(
 	"${FILESDIR}"/${PN}-1.19.0-cross-fixes.patch
 	"${FILESDIR}"/${P}-no-drm.patch
 	"${FILESDIR}"/${P}-fix-aliasing-violation.patch
+	# See bug 893604
+	# Fixed in upcoming release
+	"${FILESDIR}"/${P}-fix-url-processing.patch
```
Comment (Flowdalic) [inline app-text/mupdf/mupdf-1.21.1.ebuild:59]: This requires a revision bump.

File: app-text/mupdf/mupdf-1.21.1.ebuild [hunk 2]
```diff
@@ -154,7 +157,8 @@ src_install() {
 	fi
 
 	# Respect libdir (bug #734898)
-	sed -i -e "s:/lib:/$(get_libdir):" platform/debian/${PN}.pc || die "Failed to sed pkgconfig file to respect libdir in src_install()"
+	sed -i -e "s:/lib:/$(get_libdir):" platform/debian/${PN}.pc || \
+		die "Failed to sed pkgconfig file to respect libdir in src_install()"
```
Comment (Flowdalic) [inline app-text/mupdf/mupdf-1.21.1.ebuild:161]: nit: the control operator is usually put on the next line
```suggestion
		|| die "Failed to sed pkgconfig file to respect libdir in src_install()"
```

**Review by Flowdalic — CHANGES_REQUESTED** [review]

File: app-text/mupdf/mupdf-1.21.1.ebuild (hunk 2 above)
Comment (rndxelement) [reply app-text/mupdf/mupdf-1.21.1.ebuild]: Makes sense, fixed.

File: app-text/mupdf/mupdf-1.21.1-r1.ebuild [hunk 3]
```diff
@@ -157,7 +157,8 @@ src_install() {
 	fi
 
 	# Respect libdir (bug #734898)
-	sed -i -e "s:/lib:/$(get_libdir):" platform/debian/${PN}.pc || die "Failed to sed pkgconfig file to respect libdir in src_install()"
+	sed -i -e "s:/lib:/$(get_libdir):" platform/debian/${PN}.pc \
+		|| die "Failed to sed pkgconfig file to respect libdir in src_install()"
```
Comment (juippis) [inline app-text/mupdf/mupdf-1.21.1-r1.ebuild:161]: I often write it like:
```suggestion
	sed -i -e "s:/lib:/$(get_libdir):" platform/debian/${PN}.pc || 
		die "Failed to sed pkgconfig file to respect libdir in src_install()"
```
No need for backlash either then.

**Review by juippis — APPROVED** [review]

File: app-text/mupdf/mupdf-1.21.1-r1.ebuild (hunk 3 above)
Comment (rndxelement) [reply app-text/mupdf/mupdf-1.21.1-r1.ebuild:161]: Ah, I didn't know that. Thanks!

**Pull request closed without merging**